src/main/java/com/rocketden/main/controller/v1/RoomController.java

```diff
@@ -39,4 +41,10 @@ public RoomController(RoomService service) {
     public ResponseEntity<RoomDto> createRoom(@RequestBody CreateRoomRequest request) {
         return new ResponseEntity<>(service.createRoom(request), HttpStatus.CREATED);
     }
+
+    @PutMapping("/rooms/{roomId}/host")
+    public ResponseEntity<RoomDto> updateRoomHost(@PathVariable String roomId,
+                                                  @RequestBody UpdateHostRequest request) {
```

Comment:
Yeah based on what we talked about in standup (it might have been a while back) I think maybe we should change the join room and maybe get room endpoints to also use the path variables. I added a card for that in the backlog, but if we end up deciding to stick with room id as part of the request I can also come back and fix this if that sounds good